clients/besu/enode.sh

```diff
@@ -10,7 +10,11 @@
 set -e
 
 data='{"jsonrpc":"2.0","method":"admin_nodeInfo","params":[],"id":1}'
-TARGET_RESPONSE=$(curl -s -X POST  -H "Content-Type: application/json"  --data $data "localhost:8545" )
+while [[ "$TARGET_RESPONSE" != *enode* ]]
+do
+    TARGET_RESPONSE=$(curl -s -X POST  -H "Content-Type: application/json"  --data $data "localhost:8545" )
+    sleep 0.1
```

Comment:
```suggestion
    ((c++)) && ((c==50)) && break
    sleep 0.1
```
We could add a max retry count here, perhaps 5 seconds ?